tracer/test/test-applications/integrations/Samples.AWS.Kinesis/Program.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.Runtime;
+using Amazon.Kinesis;
+
+namespace Samples.AWS.Kinesis
+{
+    public class Program
+    {
+        private static async Task Main(string[] args)
+        {
+            // Set up AmazonSQSConfig and redirect to the local message queue instance
+            var kinesisClient = GetAmazonKinesisClient();
+
+#if NETFRAMEWORK
+            SyncHelpers.SendKinesisTasks(kinesisClient);
```

Comment:
```suggestion
            SyncHelpers.StartKinesisTasks(kinesisClient);
```

Reply:
I've done this in a87def01baf7fd2c7dfd02c43213b9203138ce0d since this is a .NET Framework code path